### PR TITLE
doc updates for COE release 1.6.001

### PR DESCRIPTION
**Pull request by sreejithgs**
Updated the Kafka deployment page with information on JSON support

### CODE REVIEW

File: docs/deploy-coe-with-Kafka.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Citrix ADC Observability Exporter with Kafka as endpoint
 
-Citrix ADC Observability Exporter is a container that collects metrics and transactions from Citrix ADC. It also transforms the data into the formats (such as AVRO) that are supported in Kafka and exports the data to the endpoint. Kafka is an open-source and distributed event streaming platform for high-performance data pipelines and streaming analytics.
+Citrix ADC Observability Exporter is a container that collects metrics and transactions from Citrix ADC. It also transforms the data into the formats (such as AVRO) that is supported in Kafka and exports the data to the endpoint. Kafka is an open-source and distributed event streaming platform for high-performance data pipelines and streaming analytics.
```
Comment (ritjaintibco) [inline docs/deploy-coe-with-Kafka.md:3]: that 'are' seems fine i think?

File: docs/deploy-coe-with-Kafka.md [hunk 2]
```diff
@@ -274,4 +276,45 @@ set analyticsprofile ns_analytics_time_series_profile -collectors metrichost_SVC
...
+                    "VPN": "all",
+                    "NGS": "all",
+                    "ICA": "all",
+                    "APPFW": "none",
+                    "BOT": "none",
+                    "VIDEOOPT": "none",
+                    "BURST_CQA": "none",
+                    "SLA": "none",
+                    "MONGO": "none"
+                },
```
Comment (ritjaintibco) [inline docs/deploy-coe-with-Kafka.md:317]: please remove this extra comma

**Review by ritjaintibco — CHANGES_REQUESTED** [review]

File: docs/deploy-coe-with-Kafka.md (hunk 2 above)
Comment (sreejithgs) [reply docs/deploy-coe-with-Kafka.md]: fixed now

File: docs/deploy-coe-with-Kafka.md (hunk 1 above)
Comment (sreejithgs) [reply docs/deploy-coe-with-Kafka.md]: The auto spell check was suggesting it. However, felt are in more accurate. so changed it.

**Review by ritjaintibco — APPROVED** [review]

**Pull request merged**